### DISCUSSION TITLE
Why doesn't hypervisor work on sideloaded UTM

### DISCUSSION THREAD

**Discussion by stossy11** [utmapp/UTM#4641] [UTM Help (iOS)] [answered]
i was wondering because i have a iPad Air 4th Gen on iPadOS 16.1 and i was looking at the releases tab and it hypervisor not on UTM.ipa so can anyone please answer my question 

Thanks
Stossy11

**Comment by osy** [comment, marked as answer]
https://docs.getutm.app/installation/ios/#summary

**Reply by Maxfusion26** [reply]
Can virtualization be enabled on something virtual JB such as Sileo or etc?

**Reply by stossy11** [reply]
I have an M1 iPad Air 5 Now Can i enable hypervisor on sideloaded UTM
